modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java

```diff
@@ -0,0 +1,166 @@
+package org.testcontainers.containers.localstack;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import org.jetbrains.annotations.Nullable;
+import org.junit.rules.ExternalResource;
+import org.rnorth.ducttape.Preconditions;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.LogMessageWaitStrategy;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static org.testcontainers.containers.BindMode.READ_WRITE;
+
+/**
+ * <p>Container for Atlassian Labs Localstack, 'A fully functional local AWS cloud stack'.</p>
+ * <p>{@link LocalStackContainer#withServices(Service...)} should be used to select which services
+ * are to be launched. See {@link Service} for available choices. It is advised that
+ * {@link LocalStackContainer#getEndpointConfiguration(Service)} and
+ * {@link LocalStackContainer#getDefaultCredentialsProvider()}
+ * be used to obtain compatible endpoint configuration and credentials, respectively.</p>
+ */
+public class LocalStackContainer extends ExternalResource {
+
+    @Nullable private GenericContainer delegate;
+    private Service[] services;
+
+    @Override
+    protected void before() throws Throwable {
+
+        Preconditions.check("services list must not be empty", services != null && services.length > 0);
+
+        final String servicesList = Arrays
+                .stream(services)
+                .map(Service::getLocalStackName)
+                .collect(Collectors.joining(","));
+
+        final Integer[] portsList = Arrays
+                .stream(services)
+                .map(Service::getPort)
+                .collect(Collectors.toSet()).toArray(new Integer[]{});
+
+        delegate = new GenericContainer("atlassianlabs/localstack:0.6.0")
```

Comment:
since this image is deprecated, maybe we should not add this module with it and replace with a recommended one?

Reply:
Good idea, this is done.